### PR TITLE
New Icon: Dahua

### LINKED ISSUE #8273: Dahua
### Brand Name

Dahua

### Website

https://www.dahuasecurity.com/

### Popularity Metric

The similarweb rank is 48,722 .See: https://www.similarweb.com/website/dahuasecurity.com/

### Official Resources for Icon and Color

None.

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by ripzerotwo**
**Issue:** closes #8273 

**Similarweb rank:** 48,722 (https://www.similarweb.com/website/dahuasecurity.com/#overview)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
Svg taken from wikimedia. Hex from Svg
Preview...
dahua

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2749,6 +2749,11 @@
             "hex": "00529B",
             "source": "https://www.daf.com/en"
         },
+        {
+            "title": "Dahua",
+            "hex": "E60012",
+            "source": "https://upload.wikimedia.org/wikipedia/commons/f/f5/Dahua_Technology_logo.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:2755]:
```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:Dahua_Technology_logo.svg"
```

File: icons/dahua.svg
Comment (adamrusted) [inline icons/dahua.svg:1]: Your path seems [incredibly complex](https://github.com/simple-icons/simple-icons/assets/1830867/82f8d634-ef7a-4a0b-ab32-d4cee3d1fe24) based on the number of nodes in it. Would you be able to simplify this down? Could you also provide examples (other than the favicon) where the `da` gets used?

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/dahua.svg
Comment (PeterShaggyNoble) [reply icons/dahua.svg:1]: From https://www.dahuasecurity.com/aboutUs/Trademark-Usage-Guideline, I would agree that the full logo would need to be used:
> **4. Do Not Alter Dahua Trademarks**
> Trademarks should be used consistently and should not be altered. For example, please do not:
> - Abbreviate the trademarks.

**Comment by PeterShaggyNoble** [comment]
We may not be able to add this at all. From their [Trademark Usage Guidelines](https://www.dahuasecurity.com/aboutUs/Trademark-Usage-Guideline):
>  **Third-party Usage of Dahua Trademarks**
> 
> Please note even when accurately describing a relationship with Dahua or support of products or services of Dahua, you should NOT assume that you have the right to use any Dahua Logos except as specifically provided by a license or other written agreement between you and Dahua.
> 
> You may only use Dahua trademarks in a manner that has been approved and authorized by Dahua in writing by an authorized representative of Dahua. Any third party shall not make unlicensed use of The Dahua Logos, such as the _[Dahau Technology]_ logo. If you are interested in obtaining a license to use the Dahua Logos or marks, contact your Dahua marketing or sales representative, or your local Dahua sales office.
>
> If you are a licensee of Dahua trademarks, your trademark license agreement that you signed with Dahua may have special trademark and logo usage guidelines different from the Guidelines set forth here. If so, please follow the special guidelines provided to you pursuant to your license agreement.
>
> Except as set forth herein, Dahua retains and reserves all rights to the Dahua Trademarks and their use, including the right to modify the Guidelines. The Guidelines (except for trademark licenses executed separately) may be amended from time to time at the discretion of Dahua and such changes will be effective ten (10) days after the changes are posted. Dahua may provide you with notice of such changes, but need not do so. You are responsible for checking this site for any changes.

**Comment by PeterShaggyNoble** [comment]
It's been brought to my attention that we are now outright rejecting requests for an contributions of icons that require permission to be used. If you or @ScapSie would like to get in touch with Dahau and request & receive that permission then we will be happy to revisit this PR.

Thanks for your efforts nonetheless, @ripzerotwo 👍🏻

**Pull request closed without merging**